src/components/Quiz/QuizWidget/QuizRadioGroup.tsx

```diff
@@ -4,7 +4,6 @@ import {
   Box,
   chakra,
   ChakraProps,
-  Circle,
   HStack,
   Stack,
   Text,
```

Comment:
Is it possible to migrate this component completely to Tw? I feel that we only need to change a few components and avoid leaving this in a hybrid situation.

Reply:
@pettinarip I started that and it led to a few complications that I didn't anticipate. For the sake of keeping scope of this PR trimmed to just the color updates, mind if I branch off and patch this up separately?

Reply:
Got it. Sure, we can tackle that separately.